cli/tests/pcluster/config/test_update_policy.py

```diff
@@ -1279,10 +1571,9 @@ def test_shared_storage_update_policy_condition_checker(
         )
     if scheduler != "awsbatch":
         cluster_has_running_login_nodes_mock.assert_called()
-        if has_running_login_nodes:
-            cluster_has_running_capacity_mock.assert_not_called()
-        else:
-            cluster_has_running_capacity_mock.assert_called()
+        cluster_has_running_capacity_mock.assert_called()
```

Comment:
Minor: Previously we were saving a call to has_running_capacity, after checking that the cluster had login nodes.

Reply:
As a consequence of the changes in the policy checker logic, we check login fleet and compute fleet independently. So, even if the there are running login nodes, we also check the compute nodes to provide a comprehensive failure reason.